coderd/workspacestats/reporter.go

```diff
@@ -31,6 +46,10 @@ type ReporterOptions struct {
 	UsageTracker          *UsageTracker
 	UpdateAgentMetricsFn  func(ctx context.Context, labels prometheusmetrics.AgentMetricLabels, metrics []*agentproto.Stats_Metric)
 
+	// DisableDatabaseStorage prevents storing stats in the database.  The
+	// reporter will still call UpdateAgentMetricsFn and bump workspace activity.
+	DisableDatabaseStorage bool
```

Comment:
```suggestion
	DisableDatabaseInserts bool
```
Minor: Suggestion to make behavior clearer.

Reply:
Ooo I like that